solutions/binary_search_solution.rb

```diff
@@ -1,3 +1,5 @@
+# Write a monkey patch of binary search:
+# E.g. [1, 2, 3, 4, 5, 7].my_bsearch(5) => 4
```

Comment:
Add a note to the problem description (make sure to update in the problem file itself as well) stating that they are not allowed to use `Array#index`, `Array#find_index`, `Array#include?` or `Array#member?` methods in their implementation.